Lib/test/test_email/test_message.py

```diff
@@ -1054,6 +1055,37 @@ def test_string_payload_with_multipart_content_type(self):
         attachments = msg.iter_attachments()
         self.assertEqual(list(attachments), [])
 
+    def test_string_payload_with_base64_cte(self):
+        msg = message_from_string(textwrap.dedent("""\
+        Content-Transfer-Encoding: base64
+
+        SGVsbG8uIFRlc3Rpbmc=
+        """), policy=policy.default)
+        self.assertEqual(msg.get_payload(decode=True), b"Hello. Testing")
+        self.assertDefectsEqual(msg['content-transfer-encoding'].defects, [])
```

Comment:
While it was very valuable to write this test as part of the bug fixing process, there are other existing tests of successful base64 decoding, so it isn't strictly necessary.  Keep it or not at your discretion, but see comment on moving the tests.